lib/Response.js

```diff
@@ -6,7 +6,9 @@ class Response {
   constructor(options) {
     options = options || {}; // eslint-disable-line no-param-reassign
 
-    this.body = options.body || {};
+    this.body = Object.prototype.hasOwnProperty.call(options, 'body')
```

Comment:
Calling in this way because of http://eslint.org/docs/rules/no-prototype-builtins warning, I can always disable for the line and call the way you suggested, but will leave that up to you